packages/fast-usdc/src/exos/settler.js

```diff
@@ -206,19 +215,19 @@ export const prepareSettler = (
         ) {
           const { mintedEarly } = this.state;
           const { value: fullValue } = fullAmount;
+          // XXX i think this only contains Advancing txs - should this be a separate store?
```

Comment:
comment for reviewers, not something to check in

Reply:
what's the alternative? pros/cons?

Reply:
It'd take some time to think of an attack vector - currently coming up short.

In general, they're pretty distinct behaviors / states so commingling didn't seem right. 

Since we don't currently have sufficient motivation to separate, I'll leave as-is for now and remove the comment.